src/user/controller/collection.ts

```diff
@@ -11,7 +11,11 @@ class UserCollectionController extends Controller {
   async get(ctx: Context) {
 
     const principalService = new services.principal.PrincipalService(ctx.privileges);
-    const users = await principalService.findAll('user');
+
+    const pageInt = parseInt(ctx.request.query.page);
+    const page = isNaN(pageInt) ? 1 : pageInt;
```

Comment:
Alternative (if it's not too code-golfy)


```suggestion
    const page = +ctx.request.query.page || 1;
```

Reply:
No I don't super care!

Reply:
I'm also a `Number()`fan though I get `+`